tests/build.gradle

```diff
@@ -250,8 +255,33 @@ gradle.projectsEvaluated {
             showStandardStreams = true
             exceptionFormat = 'full'
         }
+        maxHeapSize = "1024m" //Gradle 5.5 defaults to 512MB which is low
```

Comment:
With Gradle 5.0 it now [defaults to 512M of heap][1]. Due to this OOM was seen in test runs on Travis (which has 7 GB memory). Hence specifying the size explicitly to 1024 MB now

> All workers, including compilers and test executors, now start with 512MB of heap. The previous default was 1/4th of physical memory. Large projects may have to increase this setting on the relevant tasks, e.g. JavaCompile or Test.



[1]: https://docs.gradle.org/current/userguide/upgrading_version_4.html#rel5.0:default_memory_settings